client/src/components/Category/index.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import "./style.css";
+import axios from "axios";
+class Category extends React.Component {
+  state = {
+    result: []
+  };
+  componentDidMount() {
+    axios
+      .get("/category")
+      .then(data => {
+        return data;
+      })
+      .then(data => {
+        const resultArray = data.data.data.map(e => e);
+        this.setState({ result: resultArray });
+      });
+  }
+
```

Comment:
```suggestion
  componentDidMount() {
    axios
      .get("/category")
      .then(data => {
        const resultArray = data.data.data.map(e => e);
        this.setState({ result: resultArray });
      })
  }

```
there's no need for 
```
      .then(data => {
        return data;
      })
```
extra code you can simplify it like above.